src/__tests__/scenes/__snapshots__/RequestScene.test.tsx.snap

```diff
@@ -164,9 +164,7 @@ exports[`Request should render with loaded props 1`] = `
         </TouchableOpacity>
       </View>
     </InputAccessoryView>
-    <QrCode
-      onPress={[Function]}
-    />
+    <QrCode />
```

Comment:
Why is this being removed? On small phones this is useful to show a full size qr code.